.github/release-drafter.yml

```diff
@@ -0,0 +1,33 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'feat'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+  - title: '🧰 Maintenance'
+    label: 'chore'
```

Comment:
Is this all commitizen stuff?

Reply:
Sort of. It uses the labels of a PR to draft the notes for a release. The label of the PR is given by another action according to the branch name. The notes [here](https://github.com/BrownUniversity/terraform-gcp-project/releases) were made by this release-drafter action. For instance This branch was called `feature/create-template` on PR it was given the label `feature` on merge it will be added to the list of features in the release draft